posts/static/posts/rating.css

```diff
@@ -0,0 +1,18 @@
+ .rating {
+    margin: 15px auto;
+    width: 100%;
+    max-width: 500px;
+    border: 3px solid #b3b1b1;
+    height: 30px;
+  }
+
+  .rating_score {
+    font-weight: bold;
+  }
+
+  .ratings-block{
+      background: rgba(255, 255, 255, 0.74);
+      border-radius: 0.5rem;
+      padding: 5px;   
+  }
+ 
```

Comment:
Remove line 18 :)

Reply:
Maybe revert it back @MatanP12?
I have no idea why it still shows this error

Reply:
![image](https://user-images.githubusercontent.com/36789773/168035470-24e5bb65-16b3-4a24-b40c-77ddbe0972fe.png)

Reply:
Well it seems there was a space in line 18, solved 🙂 
@TomerNewmanPrograms @yulisuliman